### PR TITLE
Add a prefix to statsd requests

### PR DESCRIPTION
**Pull request by ametaireau**

### CODE REVIEW

File: cliquet/__init__.py [hunk 1]
```diff
@@ -71,6 +71,7 @@
     'cliquet.sentry_url': None,
     'cliquet.sentry_projects': '',
     'cliquet.statsd_url': None,
+    'cliquet.statsd_prefix': 'cliquet'
```
Comment (Natim) [inline cliquet/__init__.py]: Do you want to add the env here directly or can we use it automatically after the prefix?

Shoudn't we use the project_name directly?

Comment (ametaireau) [inline cliquet/__init__.py]: I'm not sure for the env, since I believe we have different stacks depending the env used.

Comment (ametaireau) [inline cliquet/__init__.py]: We can fallback on project name if statsd_prefix isn't defined, good idea.

Comment (Natim) [inline cliquet/__init__.py]: ok

File: docs/configuration.rst [hunk 2]
```diff
@@ -128,6 +128,7 @@ StatsD metrics can be enabled (disabled by default):
 .. code-block :: ini
 
     cliquet.statsd_url = udp://localhost:8125
+    cliquet.statsd_prefix = yourproject
```
Comment (leplatrem) [inline docs/configuration.rst]: I would mention here that this is useful only if different than project_name :)

**Pull request merged**